docs/ansible.utils.get_path_filter.rst

```diff
@@ -70,8 +70,8 @@ Parameters
                     <td>
                     </td>
                 <td>
-                        <div>The variable from which the value should be extraced</div>
-                        <div>This option represents the value that is passed to filter plugin in pipe format.</div>
+                        <div>The variable from which the value should be extraced.</div>
```

Comment:
```suggestion
                        <div>The variable from which the value should be extracted.</div>
```

Reply:
These files cannot be edited directly here in github so I'll resolve the conversations as I fix them locally.